index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+      integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+      crossorigin="anonymous" />
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <title>Book Catalog</title>
+  </head>
+
+  <body>
+    <main>
+      <section
+        class="container search-id">
```

Comment:
what is the name `search-id`? This doesn't explain anything. The class name should be self explanatory.

Reply:
change class name search-id to js-search-bookId